examples/src/main/java/io/dapr/examples/pubsub/http/SubscriberController.java

```diff
@@ -58,6 +59,27 @@ public Mono<Void> handleMessage(@RequestBody(required = false) CloudEvent<String
     });
   }
 
+
+  /**
+   * Handles a registered publish endpoint on this app adding a topic which manage to forward undeliverable messages.
+   * 
+   * @param cloudEvent The cloud event received.
+   * @return A message containing the time.
+   */
+  @Topic(name = "testingtopic", pubsubName = "${myAppProperty:messagebus}",
+      deadLetterTopic = "${deadLetterProperty:deadTopic}")
+  @PostMapping(path = "/testingtopic")

```

Comment:
This is breaking integration tests, please, use a different topic name and post mapping.